tests/functional/test_cloudsearchdomain.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import mock
+
+from tests import BaseSessionTest
+
+
+class EarlyExit(BaseException):
```

Comment:
Ah I see this a fair amount.  The following usually does the trick for me: https://github.com/boto/botocore/blob/develop/tests/functional/test_s3.py#L44. I would rather mock out the response so we do not cover up any other potential errors being thrown.